styleguide.config.js

```diff
@@ -23,6 +23,15 @@ const config = {
 	pagePerSection: true,
 	exampleMode: 'expand',
 	usageMode: 'expand',
+	compilerConfig: {
+		transforms: {
```

Comment:
These are default options, maybe we need to upgrade Styleguidist?

Reply:
Good to know I'll drop it!